requirements-cuda.txt

```diff
@@ -4,6 +4,6 @@
 # Dependencies for NVIDIA GPUs
 ray >= 2.9
 nvidia-ml-py # for pynvml package
-torch == 2.3.0
+torch == 2.3.1
 xformers == 0.0.26.post1  # Requires PyTorch 2.3.0
```

Comment:
Are these a problem?

Reply:
I believe so...we probably need to wait xformers to release a new version for PyTorch 2.3.1 :(